tests/test_sailor/test_assetcentral/test_generics.py

```diff
@@ -56,6 +64,10 @@ def test_find_functions_expect_fetch_call_args(test_object):
     objects = [instance_class({params['id_field']: x}) for x in ['test_id1', 'test_id2']]
     expected_result = params['set_class'](objects)
 
+    monkeypatch.setitem(instance_class._field_map,
+                        'integer_param1', _base.MasterDataField('integer_param1', 'integer_param1',
+                                                                query_transformer=lambda x: str(x)))
+
```

Comment:
this is basically just testing a query transformer again here. we could also simply make the integer field a regular string field.